tests/unit/sagemaker/image_uris/test_dlc_frameworks.py

```diff
@@ -385,13 +393,13 @@ def test_pytorch_eia(pytorch_eia_version, pytorch_eia_py_version):
     )
     assert expected == uri
 
-    for region, account in DLC_ALTERNATE_REGION_ACCOUNTS.items():
+    for region in ELASTIC_INFERENCE_REGIONS:
         uri = image_uris.retrieve(region=region, **base_args)
 
         expected = expected_uris.framework_uri(
             "pytorch-inference-eia",
             pytorch_eia_version,
-            account,
+            DLC_ACCOUNT,
```

Comment:
I would recommend looking up the account id from the `DLC_ACCOUNT` and `DLC_ALTERNATE_REGION_ACCOUNTS` variables, because we never know when opt-in regions or other partitions might receive EI support.

I guess something like `account = DLC_ALTERNATE_REGION_ACCOUNTS.get(region, DLC_ACCOUNT)` would be good.

Reply:
fixed

Reply:
Thanks!